src/components/Game.js

```diff
@@ -393,7 +399,14 @@ class Game extends React.Component {
             } else if ('scoreB' in json){
               scoreB += 1;
             }
-          }
+          }else if(json["textroom"] === "chatBox"){
+			remoteMessage = json["chat"]
+			remoteSender  = json["sender"]
+			document.getElementById('chatBox').innerHTML =  document.getElementById('chatBox').innerHTML +
+													'<br>'+ remoteSender + ":" + remoteMessage;
```

Comment:
Can we delete `remoteSender` and `remoteMessage` variables? And just use `json` instead?

Reply:
Yes, I deleted them